src/test/java/com/cronutils/model/time/generator/ExecutionTimeUnixIntegrationTest.java

```diff
@@ -554,4 +552,14 @@ public void fuzzyHitReturnsVerySmallIntervalDuration() {
             fail(LAST_EXECUTION_NOT_PRESENT_ERROR);
         }
     }
+
+    @Test
+    public void invalidDayInMonthCron() {
+        final CronDefinition cronDefinition = CronDefinitionBuilder.instanceDefinitionFor(CronType.UNIX);
+        final CronParser parser = new CronParser(cronDefinition);
+        final Cron myCron = parser.parse("0 0 31 2 *");
+        final ZonedDateTime time = ZonedDateTime.parse("2015-09-17T00:00:00.000-07:00");
+        final Optional<ZonedDateTime> nextExecution = ExecutionTime.forCron(myCron).nextExecution(time);
```

Comment:
will this still be a valid test when the day of month validation fix lands?

Reply:
nope, it won't. Good point. I think I can remove it then - I needed the test to verify the behavior now because I don't know how to create the infinite loop once I add the validation :)